stanford/roles/insights_deployment/tasks/main.yml

```diff
@@ -0,0 +1,159 @@
+---
+# Create Analytics Database Subnets
```

Comment:
> One of the issues with this is that I have lines of the type: cidr: "{{ DEPLOYMENT_CIDR_BASE }}.175.0/24" since there doesn't seem to be anywhere outside of cluster/defaults/main.yml to define the vars.

You should be able to refer to these via `CLUSTER_SUBNETS`, eg: for the jumpbox:
- `{{ CLUSTER_SUBNETS['jump'][0].cidr }}`
- `{{ CLUSTER_SUBNETS['jump'][1].cidr }}`